android/anc/src/main/AndroidManifest.xml

```diff
@@ -2,12 +2,37 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.smartregister.fhircore.anc">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CAMERA" />
+
     <application
+        android:name=".AncApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Fhircoreandroid" />
+        android:theme="@style/Theme.Fhircoreandroid">
+        <activity
```

Comment:
Consider using the correct theme for the Activity. For **eir** application we are using the `android:theme="@style/AppTheme.NoActionBar"`. We may need to refactor some of the shared theme colors and styles to  **engine** module